src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/BasePageObject.java

```diff
@@ -181,7 +181,8 @@ protected void scrollAndClick(WebElement element, int offset) {
   protected void scrollToElement(WebElement element) {
     try {
       ((JavascriptExecutor) driver).executeScript("var x = $(arguments[0]);"
-          + "window.scroll(0,parseInt(x.offset().top - 100));", element);
+                                                  + "window.scroll(0,parseInt(x.offset().top - 100));",
+                                                  element);
     } catch (WebDriverException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)